app/src/main/java/org/mozilla/fenix/FeatureFlags.kt

```diff
@@ -27,7 +27,7 @@ object FeatureFlags {
     /**
      * Enables WebAuthn support.
      */
-    val webAuthFeature = Config.channel.isNightlyOrDebug
+    val webAuthFeature = true
```

Comment:
There's a lint error. It wants this to be a `const`:
MayBeConst - [webAuthFeature] at /builds/worker/checkouts/src/app/src/main/java/org/mozilla/fenix/FeatureFlags.kt:30:9

Reply:
Ah heck, this is why I shouldn't try to edit a patch in the Github UI. 🙃